Examples/CaseStudies/SwiftUICaseStudiesTests/04-HigherOrderReducers-ReusableFavoritingTests.swift

```diff
@@ -82,7 +77,7 @@ final class ReusableComponentsFavoritingTests: XCTestCase {
         id: episodes[0].id, action: .favorite(.response(.failure(FavoriteError()))))
     ) {
       $0.episodes[id: episodes[0].id]?.alert = AlertState(
-        title: TextState("Favoriting failed.")
+        title: TextState(FavoriteError().localizedDescription)
```

Comment:
Can you revert this one change assuming things still pass with `"Favoriting failed."`? We prefer to assert against concrete data where possible,

Reply:
Sure! I reverted it.